src/sickle.ts

```diff
@@ -222,7 +222,16 @@ class Annotator {
   private writeNode(node: ts.Node, skipComments: boolean = false) {
     if (node.getChildCount() == 0) {
       // Directly write complete tokens.
-      this.emit(skipComments ? node.getText() : node.getFullText());
+      if (skipComments) {
+        if (node.getFullStart() < node.getStart()) {
```

Comment:
Unfortunately this doesn't actually check for a comment, so it feels wrong to pull it into a function.  I will add a comment and a TODO.